tox.ini

```diff
@@ -71,19 +69,13 @@ setenv =
     dev: WEBSOCKET_URL = {env:WEBSOCKET_URL:ws://localhost:5001/ws}
     OBJC_DISABLE_INITIALIZE_FORK_SAFETY = YES
 deps =
-    tests: -r requirements/tests.txt
-    functests: -r requirements/functests.txt
-    lint: -r requirements/lint.txt
-    {format,checkformatting}: -r requirements/format.txt
-    coverage: -r requirements/coverage.txt
-    {docs,checkdocs}: -r requirements/docs.txt
-    dev: -r requirements/dev.txt
-    dockercompose: -r requirements/dockercompose.txt
-    {env:EXTRA_DEPS:}
+    pip-tools
```

Comment:
Might we want to pin `pip-tools`? To prevent new versions of `pip-tools` from breaking things. It should be easy to do: move this to a pair of `requirements/pip_tools.{in,txt}` and in the `tox.ini` put `deps = -r requirements/pip_tools.txt`. Dependabot will then update `pip_tools.txt` for us. The only problem is that tox won't detect when `pip_tools.txt` has changed :facepalm: But maybe that's okay?